### PR TITLE
He moves east

### PR DESCRIPTION
**Pull request by xori**

### CODE REVIEW

File: src/Components/Main.css [hunk 1]
```diff
@@ -1,7 +1,7 @@
 
 .main-container {
     display: grid;
-    grid-gap: 10px;
```
Comment (xori) [inline src/Components/Main.css:4]: why the change to 5px?

File: src/Components/Main.js [hunk 2]
```diff
@@ -2,13 +2,37 @@ import React, { Component } from 'react';
...
+    const variable = 520
     return (
       <div className="main-container">
         <div className="stats">Stats</div>
-        <div className="battle">Fight fight fight!</div>
-        <div className="map">Map (x:0, y:0)</div>
-        <div className="unused">Unused</div>
+        <div className="battle">
+          <span>Fight fight fight!</span>
+          <button onClick={e => this.move()} >Move</button>
```
Comment (coltin) [inline src/Components/Main.js]: Usually good to name this "event" instead of "e". Javascript already lacks types, so it can help (especially for newer people like Adam)

**Review by coltin — APPROVED** [review]

**Comment by coltin** [comment]
Ship it! Leaving for you to merge in since I think authors of the PR should write the final commit message. Squash and Merge or Rebase and Merge are preferred to limit "Merge" commits that pollute the git history.

**Comment by xori** [comment]
I wasn't sure if GitHub would detect if I merged it locally but it doesn't seem like it does.
